app/controllers/api/automate_controller.rb

```diff
@@ -1,15 +1,14 @@
 module Api
   class AutomateController < BaseController
     def show
-      ae_browser = MiqAeBrowser.new(@auth_user_obj)
-      begin
-        resources = ae_browser.search(@req.c_suffix, ae_search_options)
-      rescue => err
-        raise BadRequestError, err.to_s
+      if browser_requested?
+        resources = collection_search
+        attributes = params['attributes'] ? %w(fqname) | params['attributes'].to_s.split(',') : nil
+        resources = resources.collect { |resource| resource.slice(*attributes) } if attributes.present?
+        render_resource :automate, :name => "automate", :subcount => resources.count, :resources => resources
+      else
+        super
       end
-      attributes = params['attributes'] ? %w(fqname) | params['attributes'].to_s.split(',') : nil
-      resources = resources.collect { |resource| resource.slice(*attributes) } if attributes.present?
-      render_resource :automate, :name => "automate", :subcount => resources.count, :resources => resources
     end
 
     def git_refresh_resource(type, id = nil, data = nil)
```

Comment:
This is fine too, I can go either way with this.  I'd stick with _refresh_ though.  Is it possible to refresh from different sources for a domain ?  i.e. triggering a git refresh vs. a refresh from disk location for example ? In which case we might need the 2 actions, or just the new _refresh_ action but with an attribute like _source_type_ = "git".  Thanks.